less/shared.less

```diff
@@ -263,10 +287,7 @@ input[type="email"] {
 
 input[type="email"] {
   padding-right: 33px;
-}
-input[name="lastname"],
-input[name="zip"] {
-  padding-left: 14px;
+  margin-top: 3px;
```

Comment:
we can just add `outline: none;` on our inputs to override whatever is adding that.